mne/encoding/feature.py

```diff
@@ -0,0 +1,341 @@
+"""A collection of classes for transforming, preprocessing, and fitting."""
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#          Jona Sassenhagen <jona.sassenhagen@gmail.de>
+#          Jean-Remi King <jeanremi.king@gmail.com>
+#
+# License: BSD (3-clause)
+import numpy as np
+from scipy import sparse
+from ..utils import warn, _reject_data_segments
+
+
+class DataDelayer(object):
```

Comment:
I assume it's the predictors (events and continuos covariates).

It should (if I'm getting this correctly) be roughly equal to this
https://github.com/mne-tools/mne-python/blob/master/mne/stats/regression.py#L329